woocommerce/woocommerce
# 50555 - Update button copy and change to outline button

### Changes proposed in this Pull Request:

Closes #50555 

Before:

<img width="428" alt="Screenshot 2024-09-25 at 9 05 52 PM" src="https://github.com/user-attachments/assets/2858de0b-020c-461b-8829-1fb58073f3fe">

After:

<img width="415" alt="Screenshot 2024-09-25 at 9 06 57 PM" src="https://github.com/user-attachments/assets/565bfba2-28c1-4ddb-82a7-446d44c12c96">




### How to test the changes in this Pull Request:

This is a visual / copy change only, so confirming that the design matches the requested design in #50555 should be sufficient.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Update button style and copy within shipping calc address form.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
